hw/lint/doc/README.md

```diff
@@ -58,10 +64,13 @@ targets:
         verilator_options:
           - "-Wall"
 ```
-Note that the setup shown above also supports Verilator lint as a fall-back option that is open source.
-The same lint target is reused however for both AscentLint and Verilator (we override the tool selection when invoking FuseSoC).
+Note that the setup shown above also supports RTL style linting with the open source tool [Verible](https://github.com/google/verible/) and RTL linting with [Verilator](https://www.veripool.org/wiki/verilator) in order to complement the sign-off lint flow with AscentLint.
+In particular, Verible lint focuses on different aspects of the code, and detects style elements that are in violation with our [Verilog Style Guide](https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md).
+
+The same lint target is reused for all three tools (we override the tool selection when invoking FuseSoC).
 Lint waivers can be added to the flow by placing them in the corresponding waiver file.
 In this example this would be `lint/aes.waiver` for AscentLint and `lint/aes.vlt` for Verilator.
+Verible does not yet support waiver files, but support for that has been provisioned in this build flow, since waiver files may be supported in the near future.
```

Comment:
I'd remove that sentence as waiver support will come very soon, and we're highly likely to forget about that sentence then :-)

Reply:
Yeah, true... removed.